apis/policies/v1alpha1/clusteradmissionpolicy_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-
+Copyright 2021.
```

Comment:
See my comment on boilerplate.go.txt, we have to update that, and regenerate with `make generate`.